rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java

```diff
@@ -153,4 +153,13 @@ private LicenseService.Iface getThriftLicenseClient() throws TTransportException
         TProtocol protocol = new TCompactProtocol(thriftClient);
         return new LicenseService.Client(protocol);
     }
+
+	public void importSpdxInformation(User sw360User) throws TException {
+		LicenseService.Iface sw360LicenseClient = getThriftLicenseClient();
+		if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+            RequestSummary deleteLicenseStatus = sw360LicenseClient.importAllSpdxLicenses(sw360User);
+        } else {
+            throw new HttpMessageNotReadableException("Unable to import All Spdx license. User is not admin");
+        }
+	}
 }
```

Comment:
Use proper indentation for the above method.

Reply:
done.